cache/compressed_secondary_cache.h

```diff
@@ -44,6 +46,16 @@ class CompressedSecondaryCacheResultHandle : public SecondaryCacheResultHandle {
 
 // The CompressedSecondaryCache is a concrete implementation of
 // rocksdb::SecondaryCache.
+// When a block is firstly Lookup from CompressedSecondaryCache, we just
```

Comment:
"firstly Lookup" gives me a feeling that no matter whether we find a block from compressed cache or not, we would insert a dummy block. Perhaps clarify it.